pkg/services/export/export_auth.go

```diff
@@ -13,58 +16,119 @@ func dumpAuthTables(helper *commitHelper, job *gitExportJob) error {
 			comment: "auth tables dump",
 		}
 
-		tables := []string{
-			"user", // joined with "org_user" to get the role
-			"user_role",
-			"builtin_role",
-			"api_key",
-			"team", "team_group", "team_role", "team_member",
-			"role",
-			"temp_user",
-			"user_auth_token", // no org_id... is it temporary?
-			"permission",
+		type statsTables struct {
+			table      string
+			sql        string
+			converters []sqlutil.Converter
+			drop       []string
 		}
 
-		for _, table := range tables {
-			switch table {
-			case "permission":
-				sess.Table(table).
-					Join("left", "role", "permission.role_id = role.id").
-					Cols("permission.*").
-					Where("org_id = ?", helper.orgID).
-					Asc("permission.id")
-			case "user":
-				sess.Table(table).
-					Join("inner", "org_user", "user.id = org_user.user_id").
-					Cols("user.*", "org_user.role").
-					Where("org_user.org_id = ?", helper.orgID).
-					Asc("user.id")
-			case "user_auth_token":
-				sess.Table(table).
-					Join("inner", "org_user", "user_auth_token.id = org_user.user_id").
-					Cols("user_auth_token.*").
-					Where("org_user.org_id = ?", helper.orgID).
-					Asc("user_auth_token.id")
-			default:
-				sess.Table(table).Where("org_id = ?", helper.orgID).Asc("id")
+		dump := []statsTables{
+			{
+				table: "user",
+				sql: `
+					SELECT user.*, org_user.role 
+					  FROM user 
+					  JOIN org_user ON user.id = org_user.user_id
+					 WHERE org_user.org_id =` + strconv.FormatInt(helper.orgID, 10),
+				converters: []sqlutil.Converter{{Dynamic: true}},
+				drop: []string{
+					"id", "version",
+					"password", // UMMMMM... for now
```

Comment:
uh

Reply:
don't want to export passwords right now... given that we can not really import that is good enough :)